src/PolyType.Roslyn/Model/DerivedTypeModel.cs

```diff
@@ -22,6 +22,11 @@ public sealed class DerivedTypeModel
     /// </summary>
     public required int Tag { get; init; }
 
+    /// <summary>
+    /// Gets a value indicating whether <see cref="Tag"/> is inferred or explicitly specified in a more stable way.
```

Comment:
I would invert the order to emphasize the meaning of `true`.
```suggestion
    /// Gets a value indicating whether <see cref="Tag"/> has been explicitly specified or inferred in a less stable way.
```